modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,13 +1,32 @@
 = Release Notes
+// Declare module attributes in pages/_partial$/_attributes-local.adoc
 include::partial$_attributes-local.adoc[]
 
+// Declare page attributes
 :natt: None at this time
 :ke: Known issues
 :fixed: Fixed at this Release
 :enh: Enhancements
 :nftr: New Features
 :api: API Changes
 
+[#latest]
+[#2-7-4]
+== 2.7.4 -- October, 2020
+
+*{fixed}*
+
+* {url-jira}/CBG-1121[*CBG-1121*] User name was not being logged for initial HTTP request
+* {url-jira}/CBG-1093[*CBG-1093*] Running compact when UseViews=true causes an infinite loop
+* {url-jira}/CBG-1003[*CBG-1003*] Remove replication stats from sg-replicate1 checkpoint hash
+* {url-jira}/CBG-885[*CBG-885*] Align channels query ORDER BY with index
+* {url-jira}/CBG-884[*CBG-884*]	Generate empty delta as {} instead of null
```

Comment:
Remove extra spaces

Reply:
Resolved in commit aac8fbace52f692bbb39e4095dd26cc578cb4709